app/main.py

```diff
@@ -2,0 +1,49 @@
+from typing import List
+
+
+class Animal:
+    alive: List["Animal"] = []
```

Comment:
It's better to use a more descriptive variable name instead of `alive`. For example, `alive_animals` would be more descriptive and make the code easier to understand.